src/styles/blocks/cards.scss

```diff
@@ -0,0 +1,72 @@
+.card {
+  padding: 32px 16px 16px;
+
+  width: 200px;
+
+  border-radius: 5px;
+  border: 1px solid $borderColor;
+  background-color: #fff;
+
+  &__image {
+    margin: 0 auto;
+    margin-bottom: 40px;
+  }
+
+  &__title {
+    color: $mainColor;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin-bottom: 4px;
+  }
+
+  &__code {
+    color: $greyColor;
+    font-size: 10px;
+    line-height: 14px;
+    margin-bottom: 16px;
+  }
+
+  &__price {
+    color: $greyColor;
+    font-size: 12px;
+    font-weight: 400;
+    line-height: 18px;
+
+    margin-top: 24px;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    margin-bottom: 16px;
```

Comment:
use `margin` in one direction only to avoid margin collapse.
If the upper block will contain `margin-bottom` - indents will not work properly